packages/chaire-lib-backend/src/utils/processManagers/__tests__/TrRoutingProcessManager.test.ts

```diff
@@ -56,7 +59,7 @@ describe('TrRouting Process Manager: start', () => {
             'trRouting4000',
             'trRouting',
             'trRouting',
-            ['--port=4000', `--osrmPort=${walkingOsrmMode.getHostPort().port}`, `--osrmHost=${walkingOsrmMode.getHostPort().host}`, '--debug=0', `--cachePath=${directoryManager.projectDirectory}/cache/test`],
+            ['--port=4000', `--osrmPort=${walkingOsrmMode.getHostPort().port}`, `--osrmHost=${walkingOsrmMode.getHostPort().host}`, '--debug=0', `--cachePath=${directoryManager.projectDirectory}/cache/test`, '--threads=2'],
```

Comment:
So this is not configurable for the main trRouting process?

Reply:
No!   We can add an issue if we want to add a parameter for this. But I would wait until we refactor configuration management